applications/EigenSolversApplication/custom_solvers/ublas_wrapper.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
```

Comment:
Can you please use the class template? (kratos/templates)
Also I suggest putting this file in `custom_utilities`